wrappers/common/common_go1.11.go

```diff
@@ -0,0 +1,26 @@
+//+build go1.11
+
+package common
+
+import (
+	"database/sql"
+
+	"github.com/honeycombio/beeline-go/trace"
+	libhoney "github.com/honeycombio/libhoney-go"
+)
+
+func addDBStatsToEvent(ev *libhoney.Event, stats sql.DBStats) {
+	ev.AddField("db.open_connections", stats.OpenConnections)
+	ev.AddField("db.connections_in_use", stats.InUse)
+	ev.AddField("db.connections_idle", stats.Idle)
+	ev.AddField("db.wait_count", stats.WaitCount)
+	ev.AddField("db.wait_duration", stats.WaitDuration)
```

Comment:
The godoc makes me think these (`WaitCount` and `WaitDuration`)are both an ever-increasing counters

> The total number of connections waited for
> The total time blocked waiting for a new connection

Is that total ... over the lifetime of the process? If so, I would leave it out.

Reply:
Yeah, I was thinking about those - they ARE over lifetime of the process, but they are super important signals and give you really good signal as to how much impact database contention is having during an incident. In normal operation for most systems, these values should be zero. For example, we could use this to trivially determine if we are running into DB contention issues due to cache filling when shepherd starts, which would be very difficult to determine using the other stats.